RequestService/RequestService.Core/Domains/Entities/UpdateRequestRequest.cs

```diff
@@ -0,0 +1,21 @@
+﻿using HelpMyStreet.Utils.Enums;
+using MediatR;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace RequestService.Core.Domains.Entities
+{
+    public class UpdateRequestRequest : IRequest
+    {
+        public string Postcode { get; set; }
```

Comment:
Realised we need to supply RequestID here, or it won't know which request to update!